### PR TITLE
feat: handle background on separate canvas

### PR DESCRIPTION
**Pull request by gpvidal**

### CODE REVIEW

File: src/SketchField.jsx [hunk 1]
```diff
@@ -257,41 +257,44 @@ class SketchField extends PureComponent {
...
-      let tempScaleY = scaleY * hfactor;
-      let tempLeft = left * wfactor;
-      let tempTop = top * hfactor;
-      obj.scaleX = tempScaleX;
-      obj.scaleY = tempScaleY;
-      obj.left = tempLeft;
-      obj.top = tempTop;
-      obj.setCoords()
-    }
+    backgroundCanvas.setWidth(offsetWidth - widthCorrection);
```
Comment (sasalatart) [inline src/SketchField.jsx]: These 2 operations can also go inside the forEach, or not?

**Review by sasalatart — APPROVED** [review]
Small comment only

File: src/SketchField.jsx (hunk 1 above)
Comment (gpvidal) [reply src/SketchField.jsx]: Yep, I'll try

Comment (gpvidal) [reply src/SketchField.jsx]: It's works :ok_hand:

**Pull request merged**